setup.py

```diff
@@ -92,7 +92,7 @@
         'pypandoc',
         # Documentation
         'sphinx>=4.3.0',
-        'sphinx-rtd-theme>=0.5.1"',
+        'sphinx-rtd-theme>=0.5.1',
```

Comment:
odd to see this in the diff since already fixed in `maint`:
```
❯ git describe
0.17.10-2-g23b376756
❯ grep rtd setup.py
        'sphinx-rtd-theme>=0.5.1',
```
but ok

Reply:
I was surprised as well but did not dive into it any further.